src-tauri/bindings/ChannelMessagePayload.ts

```diff
@@ -2,4 +2,4 @@
 import type { TextPacket } from "./TextPacket";
 import type { WaypointPacket } from "./WaypointPacket";
 
-export type ChannelMessagePayload = { text: TextPacket };
+export type ChannelMessagePayload = { text: TextPacket } | { waypoint: WaypointPacket };
```

Comment:
This change will break builds, the UI expects a `.text` field on all messages for the time being